.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,7 +8,7 @@ Please describe the changes in detail.
 
 ## Screenshots
 
-Please include screenshots, gifs, etc.
+Upload concise visuals (screenshots, gifs, etc) showing the output and impact of changes. Avoid sharing code screenshots or videos.
```

Comment:
videos are fine, even encouraged! some of us use loom extensively. code segments are fine too if they help illustrate the solution, they're just not what we mean by screenshots.

Reply:
How about this:
## Screenshots and Visuals

Upload concise visuals (screenshots, gifs, etc) demonstrating the output and impact of changes. Feel free to use videos, especially if utilizing tools like Loom. While code segments are acceptable if they enhance understanding, avoid sharing full code screenshots or videos.

Reply:
I think we could still soften the last bit, as screenshots of the code itself are fine, just usually not necessary. What about something like this:

Please include concise visuals (videos, screenshots, etc.) demonstrating the code in action and the impact of changes. Many on the team use a free tool called [Loom](https://www.loom.com) to capture and share short clips, but feel free to utilize any tool you'd like. Screenshots of the code itself can also be helpful, but are usually not necessary.

Reply:
This is good, actually I was a bit hesitant on adding a long statement like this, but if its fine then I think this is a good one, so should I replace your statement in my PR?

Reply:
If you don't mind, I think it gets at what we're looking for a bit more concisely. That said, it's like 70% your statement, anyway haha

Reply:
I also think that your statement is very clear, so if you want me to replace my statement with yours, then let me know I'll modify this in the PR

Reply:
I think it would be best